lib/openstack_query/handlers/prop_handler.py

```diff
@@ -0,0 +1,57 @@
+from enum import Enum
+from typing import Any, Optional, Set
+from custom_types.openstack_query.aliases import (
+    PropToPropFuncMap,
+    PropFunc,
+    OpenstackResourceObj,
+)
+
+
+class PropHandler:
```

Comment:
This class doesn't derive from `HandlerBase`? If it's not meant to then I'd argue the base class isn't worth it for a single deriving class

On the naming front `PropHandler` -> something like `OpenstackResourceAccessor` to describe what it's doing more

It's not a great name but I'm stumped for something better

Reply:
`HandlerBase` has two deriving classes - `ClientSideHandler` and `ServerSideHandler` 
`PropHandler` should derive it - but it only takes `prop` as a parameter (not preset as the other two)

I can remove it if you still think it should

Reply:
Ah I didn't see the other two deriving
Does it make sense to derive `PropHandler` from this or are we forcing an arbitrary relation between the two? I don't have a strong feeling either way on this :thinking:

Reply:
yea I think we should - since all three should implement `check_supported` - its just that this function takes different arguments - `check_supported` in all three does the same thing